xapian-bindings/python3/docs/examples/xapian-prepare-trainingfile.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+#
+# Simple example script demonstrating how to prepare training file for letor module.
+#
+# Copyright (C) 2019 Vaibhav Kansagara
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301
+# USA
+
+import sys
+import xapian
+import xapianletor
+
+# We require at least five command line arguments.
+if len(sys.argv) < 6:
+    print("Usage: PATH_TO_DATABASE PATH_TO_QUERY_FILE"
+          "PATH_TO_QREL_FILE PATH_TO_TRAINING_FILE MSIZE", sep='\n', file=sys.stderr)
```

Comment:
The name of the program is missing after `Usage: `.  It's common to do something like `"Usage: %s ...." % sys.argv[0]`.

The output is be missing a space:

```
Usage: PATH_TO_DATABASE PATH_TO_QUERY_FILEPATH_TO_QREL_FILE PATH_TO_TRAINING_FILE MSIZE
```

Also the `PATH_TO_` on the filenames seems a bit redundant and makes this harder to read - compare with:

```
Usage: DATABASE QUERY_FILE QREL_FILE TRAINING_FILE MSIZE
```

`sep='\n'` seems redundant when there's only one item listed.

Reply:
fixed in a fixup commit.